.github/workflows/style.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Style
+
+on:
+  push:
+    branches: [ ert3 ]
+  pull_request:
+    branches: [ ert3 ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7', '3.8']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 ert3 --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
Should we remove the select to run validation with all possible errors instead? :) I get this running locally with no errors at least:

```
flake8 ert3 --max-line-length=88 --ignore=F401,E711
```

88 as this is the line length enforced by black... Afterwards, both of the two ignored errors should be pretty straight forward to fix such that we can run it with no ignores.

Reply:
Thanks for your suggestion, I agree that its better to run it like this :) What was there was really just from the Python workflow template ( https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions) that I used as a starting point.

Reply:
I figured as much ;)